Calibration/HcalAlCaRecoProducers/test/AlCaDiJetsProducer_cfg.py

```diff
@@ -0,0 +1,35 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("MYDIJETS")
+
+process.load("Configuration.Geometry.GeometryIdeal_cff")
+process.load("Configuration.StandardSequences.MagneticField_cff")
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+from Configuration.AlCa.autoCond import autoCond
+process.GlobalTag.globaltag=autoCond['startup']
```

Comment:
please use the `condDBv2` version
The correct key for `autoCond_condDBv2` dictionary is `run2_mc`.